openproblems/tasks/dimensional_reduction/metrics/root_mean_square_error.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import scipy as sp
+from scipy import spatial
+import pandas as pd
+from ....tools.decorators import metric
+
+
+@metric(metric_name="compare_rmse", maximize=True)
+def compare_RMSE(adata, method):
+    
+    """Calculate the root mean square error (RMSE) between the full (or processed) data matrix and a list of dimensionally-reduced matrices."""
+    
+    dimensional_reduction_method = "X_" + method
+    
+    kruskel_matrix = "kruskel_matrix_" + method
+    kruskel_score = "kruskel_score_" + method
+    RMSE_calculation = "RMSE_" + method
+
+    kruskel_matrices, kruskel_scores, rmse_list = [], [], []
+    
+    if method == 'pca':
```

Comment:
The metric should not need to know which method was run, since the output will always be in `adata['dimensional_reduction_method']`

Reply:
The API readme was confusing - I clarified this such that it matches scanpy's convenient, already built-in function outputs.

Reply:
We should discuss this on monday. My approach might make more sense now that I have clarified the API. Might save me some time to get on the same page with you first, perhaps there's a better approach to be taken.